packages/@react-aria/interactions/src/useFocusVisible.ts

```diff
@@ -89,9 +89,7 @@ function handleFocusEvent(e: FocusEvent) {
   // Firefox fires two extra focus events when the user first clicks into an iframe:
   // first on the window, then on the document. We ignore these events so they don't
   // cause keyboard focus rings to appear.
-  const windowObject = getOwnerWindow(e.target as HTMLElement);
-  const documentObject = getOwnerDocument(e.target as HTMLElement);
-  if (e.target === windowObject || e.target === documentObject) {
+  if (e.target === window || e.target === document) {
```

Comment:
i wonder if FF has this issue if you're nested 2 levels deep, would we need to check this both at the top level and the first iframe?
were you able to reproduce the event flow mentioned here so you'd know what to look for in a second level of nesting?
maybe your app never has that many levels? i don't know

Reply:
Yea, I wasn't able to repro this with/without this change (the additional focus events aren't firing) so I think it's fine for our use case